### PR TITLE
 Completed week 02 homework

### PR DESCRIPTION
**Pull request by Heema5657**

### CODE REVIEW

File: javascript/javascript1/week2/homework02.js [hunk 1]
```diff
@@ -0,0 +1,78 @@
+//Exercise 1 
+function getFullName(firstname, surname, useFormalName){
+    if(useFormalName){
+        return 'Lord' +  ' ' + firstname + ' ' + surname;
+    }
+    else{
+        return firstname + ' ' + surname ;
+    }
+}
+const fullname1 = (getFullName('Heema','Negandhi', true));
```
Comment (tjshamhu) [inline javascript/javascript1/week2/homework02.js]: why the parentheses here?

File: javascript/javascript1/week2/homework02.js [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+
+// Exercise 2
+const daysInWeek =['sunday','monday','tuesday','wednesday','thursday','friday','saturday'];
+function getEventWeekday(number){
+    let d = new Date();
+    let n = d.getDay();
+    let i = (n + number) % 7;
+    return daysInWeek[i];
+}
+console.log(getEventWeekday(1));
```
Comment (tjshamhu) [inline javascript/javascript1/week2/homework02.js:23]: perfect

File: javascript/javascript1/week2/homework02.js [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+
+// Exercise 3 
+function youCreateThisFunctionName(tempearture){
```
Comment (tjshamhu) [inline javascript/javascript1/week2/homework02.js]: Hmm. If you revisited this code 5 years later and saw a function named like this, you'd have no idea what the function does. It's always good to appropriately name functions based on what they do.

File: javascript/javascript1/week2/homework02.js (hunk 1 above)
Comment (Heema5657) [reply javascript/javascript1/week2/homework02.js]: It was a mistake!

File: javascript/javascript1/week2/homework02.js (hunk 2 above)
Comment (Heema5657) [reply javascript/javascript1/week2/homework02.js:23]: Thanks

File: javascript/javascript1/week2/homework02.js (hunk 3 above)
Comment (Heema5657) [reply javascript/javascript1/week2/homework02.js]: surely made the changes.

**Pull request merged**